doc/CMakeLists.txt

```diff
@@ -488,3 +488,63 @@ add_custom_target(
   COMMAND ${CMAKE_MAKE_PROGRAM} nrf
   COMMAND ${CMAKE_MAKE_PROGRAM} nrfxlib
 )
+
+
+#
+# Add targets for checking links
+#
+
+add_custom_target(
+  nrf-linkcheck
+  COMMAND ${CMAKE_COMMAND} -E env
+  ZEPHYR_BASE=${ZEPHYR_BASE}
+  ZEPHYR_BUILD=${ZEPHYR_BINARY_DIR}
+  ZEPHYR_OUTPUT=${HTML_DIR}/zephyr
+  NRF_BASE=${NRF_BASE}
+  NRF_BUILD=${NRF_BINARY_DIR}
+  NRF_OUTPUT=${HTML_DIR}/nrf
+  KCONFIG_OUTPUT=${HTML_DIR}/kconfig
+  NRF_RST_SRC=${NRF_RST_OUT}/doc/nrf
+  ZEPHYR_RST_SRC=${ZEPHYR_BINARY_DIR}/rst/doc
+  MCUBOOT_OUTPUT=${HTML_DIR}/mcuboot
+  NRFXLIB_OUTPUT=${HTML_DIR}/nrfxlib
+  ${SPHINXBUILD} -w ${NRF_SPHINX_LOG} -N -b linkcheck ${NRF_SPHINXOPTS} ${NRF_RST_OUT}/doc/nrf ${NRF_BINARY_DIR}
+  WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
+)
+
+add_dependencies(nrf-linkcheck nrf-content nrf-doxy)
```

Comment:
Maybe have these as DEPENDS in the add_custom_target?
@tejlmand Or does that behave differently?

Reply:
yes, in this particular case there is a sligthly different behavior:
> DEPENDS
>
>    Reference files and outputs of custom commands created with add_custom_command() command calls in the same directory (CMakeLists.txt file). They will be brought up to date when the target is built.
>
>    Use the add_dependencies() command to add dependencies on other targets.
https://cmake.org/cmake/help/v3.13/command/add_custom_target.html